contracts/contracts/auction/TokenDepositLocker.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.8;
+
+import "./DepositLocker.sol";
+import "../token/TrustlinesNetworkToken.sol";
+
+/*
+  The ETHDepositLocker contract locks ETH deposits
+
+  For more information see DepositLocker.sol
+*/
+
+contract TokenDepositLocker is DepositLocker {
+    TrustlinesNetworkToken public token;
+
+    constructor(TrustlinesNetworkToken _trustlinesNetworkToken) public {
```

Comment:
I think it is weird that the contract now has both a `constructor` and an `init`, I feel like it is less obvious to work with

Reply:
Changed to only use init.